tutorials/grid_coordinates.py

```diff
@@ -0,0 +1,346 @@
+"""
+.. _grid_coordinates:
+
+Gridding coordinates
+====================
+
+Grid coordinates in Verde are used to create points on a regularly spaced grid
+that are then used in the spline method to interpolate between sample data
+points. The grid can be specified either by the number of points in each
+dimension (the *shape*) or by the grid node spacing.
+
+Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`
+function. It creates a set of regularly spaced points in both the west-east and
+south-north directions, i.e. a two-dimensional spatial grid.
+
+First let's create a region that is 1000 units west-east and 1000 units south-
+north, and we will set an initial spacing to 100 units.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import verde as vd
+from matplotlib.patches import Rectangle
+
+# create the bounds for the region and set the spacing for the grid points
+spacing = 100
+west, east, south, north = 0, 1000, 0, 1000
+region = (west, east, south, north)
+
+# create the grid
+square_region_east, square_region_north = vd.grid_coordinates(

```

Comment:
I really like that you're using nice and descriptive variable names! In this case, though, it might distract the reader as to what these actually mean. I would suggest renaming them to `easting` and `northing` instead. What do you think?